docs/source/common_api_flows.rst

```diff
@@ -2,66 +2,66 @@ Common API workflows
 --------------------
 
 There are many genomics-related APIs documented at
-`cloud.google.com/genomics/v1beta2/reference <https://cloud.google.com/genomics/v1beta2/reference>`_.
+`cloud.google.com/genomics/reference/rest/ <https://cloud.google.com/genomics/reference/rest/>`_.
 
-Of the available calls, there are some very common patterns that can 
-be useful when developing your own code. 
+Of the available calls, there are some very common patterns that can
+be useful when developing your own code.
 
-The following sections describe these workflows using plain URLs and 
+The following sections describe these workflows using plain URLs and
 simplified request bodies. Each step should map 1-1 with all of the auto-generated client libraries.
 
 
 Browsing read data
 ~~~~~~~~~~~~~~~~~~
 
-* ``GET /datasets``                                    
-  
+* ``GET /datasets``
+
   List all available datasets that a current user has access to. (Or all public datasets when not using OAuth)
   Choose one datasetId from the result.
```

Comment:
"Or" seems to be the capitalized portion of the second sentence, and does not end in a punctuation, but "Choose" also seems to be capitalized.  Maybe a split into two sentences or having one being part of the other are two possibilities.

Also the API does not seem to coincide with the documentation, since a GET should be sufficient to perform the query, but via the API Explorer it throws an error regarding `projectId`.  Below is the full screenshot:

![get_dataset_error](https://cloud.githubusercontent.com/assets/6555937/16096201/ecf6fe7e-3315-11e6-9065-5c96deed5a97.png)